docs/web-testing-library/api.mdx

```diff
@@ -0,0 +1,74 @@
+---
+id: api
+title: API
+---
+
+Several utilities are provided for dealing with asynchronous code. These can be
+useful to wait for an element to appear or disappear in response to an event,
+user action, timeout, or Promise. (See the
+[guide to testing disappearance](guide-disappearance.mdx).)
```

Comment:
Does this link work?

Reply:
This works, it redirects to https://deploy-preview-1204--testing-library.netlify.app/docs/guide-disappearance